lib/jailed.js

```diff
@@ -29,12 +29,22 @@ if (__is__node__) {
     __jailed__path__ = __dirname + '/';
 } else {
     // web
-    var scripts = document.getElementsByTagName('script');
-    __jailed__path__ = scripts[scripts.length-1].src
-        .split('?')[0]
-        .split('/')
-        .slice(0, -1)
-        .join('/')+'/';
+    if (doucment.currentScript) {
```

Comment:
That looks like a typo, would be a good idea to create a test to catch these things